### PR TITLE
Wizard shows sessions in preferred timezone (from service config)

### LINKED ISSUE #3: Respect service timezone configuration from display options
### Problem
Now wizard doesn't respect configuration of service timezone that is set in the "Display Options" setting. But it should. 

### Solution
Depending on service timezone config (`useCustomerTimezone`) show sessions in preferred timezone (in customer's or in service's timezone).

### PR DESCRIPTION
**Pull request by romalytvynenko**
fixes #3

### CODE REVIEW

File: src/Core/components/CfEddbkWizard.js [hunk 1]
```diff
@@ -175,6 +168,19 @@ export default function (store, bookingDataMap, TranslateCapable, MapBookingFiel
         }
       },
 
+      /**
+       * @since [*next-version*]
+       *
+       * @property {string|null} timezone Name of timezone in which sessions will be displayed.
+       */
+      timezone () {
+        const browserTimezone = this.moment.tz.guess()
```
Comment (XedinUnknown) [inline src/Core/components/CfEddbkWizard.js]: Just like before, the component should not determine its own timezone. The local timezone should be determined outside of the component, and passed to it.

File: src/Core/utils/services.js [hunk 2]
```diff
@@ -25,6 +25,26 @@ export default function (dependencies) {
...
+    /**
+     * Function for creating datetimes in timezone.
+     *
+     * @since [*next-version*]
+     *
+     * @param {Container} container DI Container.
+     *
+     * @return {CreateDatetimeFunction}
+     */
+    createDatetime (container) {
```
Comment (XedinUnknown) [inline src/Core/utils/services.js]: Didn't we have this or similar logic somewhere else?

File: src/Core/components/CfEddbkWizard.js (hunk 1 above)
Comment (romalytvynenko) [reply src/Core/components/CfEddbkWizard.js]: I want to clarify, that this component is application-level component. So technically timezone detecting happens not in components level, but on application level. Or I'm missing something?

File: src/Core/utils/services.js (hunk 2 above)
Comment (romalytvynenko) [reply src/Core/utils/services.js]: Yes, we have similar logic of creating datetime for dashboard's JS application.

**Pull request merged**